lib/ansible/plugins/loader.py

```diff
@@ -1145,7 +1145,7 @@ def _configure_collection_loader():
         warnings.warn('AnsibleCollectionFinder has already been configured')
         return
 
-    finder = _AnsibleCollectionFinder(C.config.get_config_value('COLLECTIONS_PATHS'), C.config.get_config_value('COLLECTIONS_SCAN_SYS_PATH'))
+    finder = _AnsibleCollectionFinder(C.COLLECTIONS_PATHS, C.COLLECTIONS_SCAN_SYS_PATH)
```

Comment:
don't make this change, we eventually want to phase out the constants in favor of the function calls.

Reply:
afaict only the constants are currently templated, `get_config_value` doesn't get templated.

Reply:
@bcoca as @sivel said, the `get_config_value` is not handling templates, only the constants do. See the [screenshot](https://user-images.githubusercontent.com/2292035/160599914-887296a0-a3ba-49eb-81f2-b3b0f4246f2b.png) on https://github.com/ansible/ansible/pull/77322#issuecomment-1081743289.

Personally, I would prefer to keep this change as is, and then open a separate PR, once this one is accepted, to rewrite the access of the configuration settings by providing a method (possibly by extending `get_config_value`) that is both dynamic like `get_config_value`, but also supports templating like the constants. Separate PR, because to me the modification (and code-wide usage update) of the config access seems to be too large to role into the PR about getting ANSIBLE_HOME working.

Reply:
true, had meant to fix that a  while ago #77483

Reply:
👍🏻  In that case, I'll wait for the PR to land in devel, rebase, and switch (back) to `get_config_value` in all places.